tools/test.py

```diff
@@ -130,8 +118,11 @@ def main():
                                  args.gpu_collect)
 
     rank, _ = get_dist_info()
-    eval_cfg = cfg.get('evaluation', {})
+    eval_cfg = cfg.get('evaluation', args.eval_options)
+    eval_cfg.update(dict(metric=args.metrics))
     if rank == 0:
+        if args.work_dir is not None:
+            mmcv.mkdir_or_exist(osp.abspath(args.work_dir))
```

Comment:
If `args,work_dir` is None, will the L126 throws errors?

Reply:
fixed